src/HIE/Bios/Cradle.hs

```diff
@@ -334,14 +334,14 @@ directCradle wdir args =
 
 -- | Find a cradle by finding an executable `hie-bios` file which will
 -- be executed to find the correct GHC options to use.
-biosCradle :: FilePath -> Callable -> Maybe Callable -> Cradle a
-biosCradle wdir biosCall biosDepsCall =
+biosCradle :: FilePath -> Callable -> Maybe Callable -> Maybe FilePath -> Cradle a
+biosCradle wdir biosCall biosDepsCall mbGhc =
   Cradle
     { cradleRootDir    = wdir
     , cradleOptsProg   = CradleAction
         { actionName = Types.Bios
         , runCradle = biosAction wdir biosCall biosDepsCall
-        , runGhcCmd = runGhcCmdOnPath wdir
+        , runGhcCmd = \args -> readProcessWithCwd wdir (fromMaybe "ghc" mbGhc) args ""
```

Comment:
Would it make sense to canonicalize the path if it is relative?

Reply:
Good question, I'm not sure. The path is only ever used in this call to `readProcess`, so I think it probably doesn't matter much.